pom.xml

```diff
@@ -155,6 +155,7 @@
                             <goal>single</goal>
                         </goals>
                         <configuration>
+                            <appendAssemblyId>true</appendAssemblyId>
```

Comment:
Why do we include this change?

Reply:
The assembly plugin option is changed to True to avoid build warnings.